ruby-gem/Rakefile

```diff
@@ -11,6 +11,19 @@ end
 def build
   test_server_dir = File.join(find_server_repo_or_raise, "server")
 
+  if test_server_version != Calabash::Android::VERSION
+    raise %Q{\033[31m
```

Comment:
We should not use colors here unless we check for Windows env.

Reply:
I also thought so, but it was used several lines above and I didn't want to break the pattern :)